modules/nodes-cluster-resource-override-deploy-cli.adoc

```diff
@@ -7,7 +7,11 @@
 [id="nodes-cluster-resource-override-deploy-cli_{context}"]
 = Installing the Cluster Resource Override Operator using the CLI
 
-You can use the {product-title} CLI to install the Cluster Resource Override Operator to help control overcommit in your cluster.
+You can use the {product-title} CLI to install the Cluster Resource Override Operator to help control overcommit in your cluster. 
+
+ifndef::openshift-rosa,openshift-dedicated[]
+By default, the installation process creates a Cluster Resource Override Operator pod and two Cluster Resource Override pods on the worker nodes. You can move these pods to other nodes, such as an infrastructure node, as needed. Infrastructure nodes are not counted toward the total number of subscriptions that are required to run the environment. For more information about infrastructure nodes, see "Creating infrastructure machine sets."
```

Comment:
One more thing, is the defaulting to worker nodes something you experienced on ROSA? Why are we saying control plane nodes later, and worker nodes here?

Reply:
@maxcao13 That is the behavior I saw with the system you set up for me. Is that not the case? 

The reference to the control plane is a change that I had missed. Since fixed.

Reply:
I'll try again to make sure, but it _should_ be defauling to master (control plane) nodes if you don't specify an override: https://github.com/openshift/cluster-resource-override-admission-operator/blob/a758ea43f197f3ab36f97b51154fff3a45cfbf88/pkg/asset/deployment.go#L15

Reply:
The `clusterresourceoverride-operator-###` pod should deploy to control plane?  Would the `clusterresourceoverride-###` pods deploy to workers?

Reply:
The `clusterresourceoverride-###` pods should default schedule to nodes with the `"node-role.kubernetes.io/master": ""` label/role. 
The `clusterresourceoverride-operator-###` I'm not completely sure, but I think it will schedule on the most compatible node (i.e. there's a lot of factors to decide which node it'll be running on), but there's it shouldn't always default to worker or master or infra nodes.